src/pages/Manager.vue

```diff
@@ -35,25 +35,29 @@
 			<button class="modal-close is-large" aria-label="close"
 			        @click="showRor2IncorrectDirectoryModal = false"></button>
 		</div>
-		<ModalCard :is-active="fixingPreloader" @close-modal="closePreloaderFixModal" :can-close="true">
+		<ModalCard :is-active="isValidatingSteamInstallation" @close-modal="closeSteamInstallationValidationModal" :can-close="true">
 			<template v-slot:header>
-				<h2 class='modal-title'>Attempting to fix preloader issues</h2>
+				<h2 class='modal-title'>Attempting to fix issues with game files</h2>
 			</template>
 			<template v-slot:body>
 				<div class='notification is-warning'>
-					<p>You will not not be able to launch the game until Steam has verified the integrity of the
-						game.
+					<p>
+						You will not not be able to launch the game until
+						Steam has verified the integrity of the game files.
 					</p>
 				</div>
-				<p>Steam will be started, and will attempt to verify the integrity of {{ activeGame.displayName }}.</p>
+				<p>
+					Steam will be started, and will attempt to verify the
+					integrity of {{ activeGame.displayName }}.
+				</p>
 				<br/>
-				<p>Please check the Steam window for validation progress. If the window has not yet appeared, please be
-					patient.
+				<p>
+					Please check the Steam window for validation progress.
+					If the window has not yet appeared, please be patient.
 				</p>
 			</template>
 			<template v-slot:footer>
-				<button v-if="dependencyListDisplayType === 'view'" class="button is-info"
```

Comment:
This was interesting (being tied to `dependencyListDisplayType`), I wonder if it caused any issues before?

Reply:
The issue was that the buttons were not shown on the preloader fix modal and export-as-code modal. It wasn't a big deal as they were just "ok, close the modal" type of buttons and user had other means to close the modals.